030-Concepts/080-pricing.mdx

```diff
@@ -7,26 +7,219 @@ slug: concepts/pricing
+- **File attachment storage:** 2 GB for storing file attachments. Beyond that, unlimited storage for storing file attachments is available, in which you only pay for the additional storage you use.
+- **AI queries/month:** Includes 250 AI queries monthly. You can scale up to unlimited queries, paying only for the usage beyond the initial 250 queries.
+- **Support:** Support outliined in SLAs for guaranteed support responses and resolution times, in addition to community support so you can engage with community forums for assistance and discussions.
+
+### Enterprise plan
+
+The Enterprise plan is designed for large organizations that require specific, large-scale solutions. Xata offers customized solutions, personalized support, and discounts based on volume to maximize functionality and efficiency. Our plans are adaptable, allowing changes or enhancements in line with your project's evolution. Our pricing is straightforward, with no concealed charges; you're billed only for the services you use.
+
+**The Enterprise plan is ideal for:**
+  - Large organizations
+  - High-scale enterprises
+  - Specialized projects
+
+##### Features
+- **Database branches:** Unlimited database branches to allow for expanded data structuring and management without limitations.
+- **Availability:** Multiple availability zones to ensure projects are hosted in different zones and have improved reliability.
+- **Data storage:** Includes 15 GB of space for project-related data, and beyond that, unlimited storage in which you only pay for the additional storage you use.
+- **Search engine storage:** 15 GB for searching and storing results. Beyond that, additional search engine storage is available, in which you only pay for the additional storage you use. You can use this space for enabling full-text search. Search engine storage currently bundles storage on disk with I/O rates and compute costs. 
+- **File attachment storage:** 2 GB for storing file attachments. Beyond that, unlimited storage for storing file attachments is available, in which you only pay for the additional storage you use.
+- **AI queries/month:** Includes 250 AI queries monthly. You can scale up to unlimited queries, paying only for the usage beyond the initial 250 queries.
+- **Integrations:** Integration with AWS to connect your existing AWS account for enhanced control and security.
+- **Security:** Single Sign-On (SSO) for secure and efficient access management.
+- **Support:** Dedicated support to receive support tailored to your projects specific needs and challenges.
+
+
+### Change plans
+
+Visit the [pricing page](https://xata.io/pricing) to upgrade from the Free to Pro plan. For the Enterprise plan, contact us.
+
+### Pricing summary
```

Comment:
I tend to agree with @snide that we have one source of truth for pricing features. It looks great, and I get why it would make sense in the docs, but I'm not sure if we need an additional table. I'd use the docs as a place to go when you want more details outside of the pricing page. It's probably not worth repeating for the maintenance / synchronization burden alone. When we have dedicated instances, we have an opportunity to provide a lot more detail around the consumption metrics that is simply too much information to live on the pricing page.